src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```
@@ -33,18 +33,27 @@ import {
   SizeEnum,
   TextFieldVariantEnum,
   ResourcesTabsEnum,
-  UpdateQuizParams,
-  CategoryNameInterface
+  CategoryNameInterface,
+  ServiceFunction
 } from '~/types'
 import { getErrorMessage } from '~/utils/error-with-message'
 import { styles } from '~/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.styles'
 
+interface CreateQuizParams {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)